src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
The `deprecated_template` is actually optionnal (as in the XML), you can use only `deprecated` which is basically a flag.

In XML, there wasn't any point of having a boolean attribute (as it was the case before I added the message customization) *and* a tag, while only a tag could do the trick. The thing is, even if the tag has a `xsd:string` type, it should be able to accept no value at all (its only presence marks the service as deprecated) : `<deprecated />` (then the message template will be the generic default one) .

Having two members in the yaml (and the php too) allows us to override the flag or even use the default message without specifying the `deprecated_template` part, and just use the flag part. But as I noted in the PR, I still haven't found how to override this deprecation in XML...